contracts/core/OTCWrapperV2.sol

```diff
@@ -0,0 +1,1046 @@
+        address collateral;
+        // option type the vault is selling
+        bool isPut;
+        // option strike price
+        uint256 strikePrice;
+        // option expiry timestamp
+        uint256 expiry;
+        // order premium amount in USDC (with USDC decimals)
+        uint256 premium;
+        // order notional in USD (with 6 decimals)
+        uint256 notional;
+        // buyer address
+        address buyer;
+        // seller address
+        address seller;
+        // id of the vault
+        uint256 vaultID;
+        // otoken address (with 8 decimals)
+        address oToken;
+        // timestamp of when the order was opened
+        uint256 openedAt;
+        // order size (with 8 decimals)
+        uint256 size;
+    }
+
+    // struct defining permit signature details
+    struct Permit {
+        // permit amount (with its respective token decimals)
+        uint256 amount;
+        // permit deadline
+        uint256 deadline;
+        // permit account
+        address acct;
+        // v component of permit signature
+        uint8 v;
+        // r component of permit signature
+        bytes32 r;
+        // s component of permit signature
+        bytes32 s;
+    }
+
+    // struct defining the upper and lower boundaries of USD notional for an asset
+    struct MinMaxNotional {
+        // minimum USD notional value allowed (with 6 decimals)
+        uint256 min;
+        // maximum USD notional value allowed (with 6 decimals)
+        uint256 max;
+    }
+
+    ///@dev mapping between an asset address to a struct consisting of uint256 min and a uint256 max which will be its notional floor and cap allowed
+    mapping(address => MinMaxNotional) public minMaxNotional;
+
+    ///@notice mapping between order id and order details
+    mapping(uint256 => Order) public orders;
+
+    ///@notice mapping between order id and order status
+    mapping(uint256 => OrderStatus) public orderStatus;
+
+    ///@notice mapping between a Market Maker address and its whitelist status
+    mapping(address => bool) public isWhitelistedMarketMaker;
+
+    ///@notice mapping between an asset address and its corresponding fee
+    mapping(address => uint256) public fee;
+
+    ///@notice mapping between acct and list of all successful orders
+    mapping(address => uint256[]) public ordersByAcct;
+
+    ///@notice mapping between asset and their allowed maximum price deviation between place order and execute time (percentage with 2 decimals)
+    mapping(address => uint256) public maxDeviation;
+
+    ///@notice mapping between an asset address and its corresponding unwind fee
+    mapping(address => uint256) public unwindFee;
+
+    ///@notice buffer time between last allowed unwind moment and expiry
+    uint256 public unwindBufferDuration;
+
+    /// @notice Unwind Permit interface
+    UnwindPermitInterface public immutable UNWIND_PERMIT;
+
+    // struct defining unwind permit signature details
+    struct UnwindPermit {
```

Comment:
so even adding struct (just sourcecode) to above struct section will screw up the storage slot?

Reply:
Was added at the end mostly for safety but we can look into it